src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/AssemblyBasedCallerArgumentCollection.java

```diff
@@ -25,6 +25,7 @@
 
     public static final String MIN_BASE_QUALITY_SCORE_LONG_NAME = "min-base-quality-score";
     public static final String SMITH_WATERMAN_LONG_NAME = "smith-waterman";
+    public static final String ALLELE_EXTENSION_LONG_NAME = "allele-informative-reads-overlap-radius";
```

Comment:
I understand that this is technically a one-dimensional radius, but I would prefer radius to apply only in dimensions d = 2 and higher.  IMO, "margin" is clearer.

Reply:
done.